app/src/main/res/layout/topic_summary_view.xml

```diff
@@ -10,59 +11,88 @@
   </data>
 
   <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="140dp"
+    android:layout_width="match_parent"
     android:layout_height="160dp"
```

Comment:
Make this dynamic. Consider the case when text size is increased from settings, then this static size will create issues.

Reply:
Removed static height and kept it as wrap_content.